README.md

```diff
@@ -124,15 +124,32 @@ Couching during an in progress game is not allowed. Please stick to the rules.
 This is the recommended ruleset for the offcial league matches:
 
 - Stocks: 3
-- Time limit: 8 min.
-- Hazards: off.
-- Miis: Any moveset.
-- Game one striking: (p1 - p2 - p2 - p1). In case both players don’t want to strike, the default starter stage will be **Smashville**.
+- Time limit: 7 min.
+- Final Smash Meter: Off.
+- Spirits: Off.
+- Damage Handicap: Off.
+- Stage Selection: Anyone.
+- Items: Off and None.
+- First to: 1 Win.
+- Stage Morph: Off.
+- Stage Hazards: Off.
+- Team Attack: On.
+- Launch Rate: 1.0x.
+- Underdog Boost: Off.
+- Pausing: On.
+- Score Display: Off.
+- % Show Damage: Yes.
+- Custom Balance: Off.
+- Echo Fighters: Separate.
+- Radar: Big.
+- Allowed Controllers: Nintendo GameCube, Nintendo Pro Controller, Nintendo Joy-Con, SmashBox.
+- Mii Fighters: All moveset combinations are legal.
+- Game one striking: (p1 - p2 - p2 - p1). In case both players don’t want to strike, the default starter stage will be **Pokémon Stadium 2**. You may not play on a stage that is not on the available stages list. This will result in a double disqualification/penalization.
 
 If case of time out, the winner will be decided by the winner screen. In case of a tie, the winner will be the player with less damage at the end of the match. In case of a tie of damage or double KO, the players will play a one stock 3 minutes round on the same stage to decide the winner. In case of a player SD, the winner will be decided by the winner screen.
```

Comment:
shouldn't that line start with `In case of time out` instead of `If case of time out`?.

Reply:
Sounds good, I'll change it.